sqlglot/optimizer/optimize.py

```diff
@@ -0,0 +1,11 @@
+from sqlglot.optimizer.qualify_tables import qualify_tables
+from sqlglot.optimizer.qualify_columns import qualify_columns
+
+
+def optimize(expression, schema=None, db=None, catalog=None):
```

Comment:
can we put this in __init__ so that we can do from sqlglot.optimizer import optimize